src/interpreter/executors/instruction.rs

```diff
@@ -36,11 +36,13 @@ where
         // Should execute `lo` only if there is no rupture in the flow - that means
         // either a breakpoint or some instruction that would skip `lo` such as
         // `RET`, `JI` or `CALL`
-        if self.registers[REG_PC] == pc && state.should_continue() {
+        let result = if self.registers[REG_PC] == pc && state.should_continue() {
```

Comment:
Why was `result` added?

Reply:
This is to facilitate debug and readability. If we just return complex expressions without assigning to a variable, then we need to inspect the frame pointers in gdb - having an assigned variable is zero runtime overhead and will give us that readability for free

Reply:
cool!